.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+    branches:    
+    - '**'
+    - '!**Anca**'
+    - '!**laura**'
+  pull_request:
```

Comment:
why not? this is capstone...

Reply:
if Anca or Laura commit anything the their branches it will launch the CI for Angular and fail since they don't have the folder. If they add it, the checks will starts taking several minutes instead of seconds.

Reply:
After we all finish with the webapp we can remove this rule.

Reply:
if my understanding correct this file is in your branch already, and did not failed any checks on other branches yet.. now that it propagates to the master branch, it will be still not be in the branches w/o the files this is applied to, so IMO we are safe to proceed w/o the exclusion patterns. Also if it is tmp, and we could not (so not in the current case) a TODO(user/bugid): remove these once... note would be needed in order to not to forget them..

Reply:
As I understood from using this stuff, the config files are global. If Anca commits something to her dev branch, the Angular CI will still trigger, even though she doesn't have the workflow file in her branch.